R/hydromad.R

```diff
@@ -194,8 +435,8 @@ print.hydromad <-
     if (!is.null(x$rfit)) {
       cat(
         "Routing fit spec.:",
-        toString(deparse(x$rfit, control = c(), width.cutoff = 500),
-          width = getOption("width")
+        toString(deparse(x$rfit, control = c(), width = 500),
```

Comment:
This looks like an accidental reversion?

Reply:
This should have been dealt with in #132 but I'll change here as well just in case it does something odd....

Reply:
I suspect that @josephguillaume is right and the old version crept back in. Easily happens with merge conflicts etc.

Reply:
No problems, it's been added back in.